composer.json

```diff
@@ -24,20 +24,20 @@
         "fakerphp/faker": "^1.10",
         "myclabs/deep-copy": "^1.10",
         "sebastian/comparator": "^3.0 || ^4.0 || ^5.0",
-        "symfony/property-access": "^5.4 || ^6.0",
-        "symfony/yaml": "^5.4 || ^6.0"
+        "symfony/property-access": "^5.4 || ^6.0 || ^7.0",
```

Comment:
What about changing `^6.0` to `^6.4` in order to only support the LTS versions and the last release?

It may be done on another PR though.

Reply:
@theofidry What do you prefer?

Reply:
I think it's a good idea, but definitely in a different PR. I think it would also make sense to drop 5.4, maintaining 3 different versions is too much IMO